packages/middleware/src/handlers/prepareApiFunction/index.ts

```diff
@@ -1,11 +1,10 @@
-import type { RequestHandler } from "express";
 import { IntegrationsLoaded, MiddlewareContext } from "../../types";
 import { getApiFunction } from "./getApiFunction";
 
 export function prepareApiFunction(
   integrations: IntegrationsLoaded
-): RequestHandler {
-  return (req, res, next) => {
+): (req, res, next) => Promise<any> {
+  return async (req, res, next) => {
```

Comment:
I don't like this hack in here ;p I'd go with https://www.npmjs.com/package/express-async-handler and avoid defining the return type, then express will validate if it returns what's expected.
```suggestion
) {
  return asyncHandler(async (req, res, next) => { ... });
```

Reply:
You could also try without the lib and without the Promise<any> in return type